### ISSUE TITLE
Network file playing

### ISSUE THREAD

**Issue by tanguyantoine** [zmxv/react-native-sound#35]
Maybe be inspired by cordova plugins ?

for iOS :
https://github.com/apache/cordova-plugin-media/blob/master/src/ios/CDVSound.m

for Andoid :
https://github.com/apache/cordova-plugin-media/tree/master/src/android

Based on Apple documentation, to implement this feature AVAudioPlayer must be switched to AVPlayer

**Comment by cinder92** [comment]
👍

**Comment by GShotwell** [comment]
Thanks for the great library. I'm thinking of adding this to a project but I'm wondering if there is a plan for playing Android network files? Currently we are using react-native-video to play streaming audio files, but i would prefer this library.

**Comment by qpowell** [comment]
@GShotwell it was mentioned in #68 , but I'm not sure what the status of it is. IMO, this is the most fully featured library for dealing with audio in react-native and it'd be *really* great to see that feature parity there between the platforms. I wish there was more that I could do to contribute

**Comment by Smileo** [comment]
@zmxv can you summarize about current status the Network file playing implementation for Android?
Thanks.

**Comment by zmxv** [comment]
@Smileo Network file playing for Android would be a great feature, but I'm not actively working on it right now. PRs are always welcome.

**Comment by paulmelnikow** [comment]
I think this has been implemented. Please correct me if I'm wrong!
